examples/create_new_sticker_set/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"embed"
+	"fmt"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/go-telegram/bot"
+	"github.com/go-telegram/bot/models"
+)
+
+// Send any text message to the bot after the bot has been started
+
+func main() {
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt)
+	defer cancel()
+
+	opts := []bot.Option{
+		bot.WithDefaultHandler(handler),
+		bot.WithSkipGetMe(),
+	}
+
+	b, err := bot.New(os.Getenv("EXAMPLE_TELEGRAM_BOT_TOKEN"), opts...)
+	if nil != err {
+		// panics for the sake of simplicity.
+		// you should handle this error properly in your code.
+		panic(err)
+	}
+
+	user, err := b.GetMe(ctx)
+	if nil != err {
+		// panics for the sake of simplicity.
+		// you should handle this error properly in your code.
+		panic(err)
+	}
+	botUsername = user.Username
```

Comment:
Looks like `bot.Bot` can store such info, do not you think so?

Maybe create an issue?
But what to do with `skip-get-me` option? No get me, no info.